modules/op-release-notes-1-14.adoc

```diff
@@ -350,3 +350,15 @@ With this update, {pipelines-title} General Availability (GA) 1.14.4 is availabl
 * Before this update, if a user or an {pipelines-shortname} controller used the {pipelines-shortname} API to modify a pipeline run that was in the process of being started by {pac},  {pac} could stop and the log contained "panic" messages. With this update, the pipeline being started by {pac} can be modified concurrently.
 
 * Before this update, in {pac}, a concurrency limit setting of `0` was not interpreted as disabling the concurrency limit. With this update, a concurrency limit setting of `0` disables the concurrency limit.
+
+[id="release-notes-1-14-5_{context}"]
+== Release notes for {pipelines-title} General Availability 1.14.5
+
+With this update, {pipelines-title} General Availability (GA) 1.14.5 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-14-5_{context}"]
+=== Fixed issues
+
+* Before this update, when you used the web console and clicked a pipeline in the overview page, the pipeline details page did not contain information about tasks in the pipeline. With this update, when you click a pipeline in the overview page, the pipeline details page contains the required information.
+
+* Before this update, when you configured Tekton Chains to disable storing OCI artifacts by setting an empty `artifacts.oci.storage` value in the `TektonConfig` CR, the configuration did not work and Tekton Chains attempted to store the artifacts and logged a failure in the `chains.tekton.dev/signed` annotation. With this update, when you set an empty `artifacts.oci.storage` value in the `TektonConfig` CR, Tekton Chains does not attempt to store OCI artifacts.
```

Comment:
```suggestion
* Before this update, when you configured {tekton-chains} to disable storing OCI artifacts by setting an empty `artifacts.oci.storage` value in the `TektonConfig` CR, the configuration did not work and {tekton-chains} attempted to store the artifacts and logged a failure in the `chains.tekton.dev/signed` annotation. With this update, when you set an empty `artifacts.oci.storage` value in the `TektonConfig` CR, {tekton-chains} does not attempt to store OCI artifacts.
```

Reply:
@mramendi , Release Notes are not intended to be reusable content and reflect the current state of the item being discussed. However, using the AsciiDoc attribute MAY contribute to accuracy, so I suggest it. Totally your call, sir.